site/app/controllers/student/TeamController.php

```diff
@@ -422,4 +422,8 @@ public function showPage($gradeable_id) {
         $users_seeking_team = $this->core->getQueries()->getUsersSeekingTeamByGradeableId($gradeable_id);
         $this->core->getOutput()->renderOutput(array('submission', 'Team'), 'showTeamPage', $gradeable, $team, $teams, $lock, $users_seeking_team);
     }
+    public function getTeamIds($id, $gradeable_id) {
+        $id = $this->core->getQueries()->getTeamByGradeableAndUser($gradeable_id, $id)->getId();
```

Comment:
Move this into the controller method for the AutoGradingView and pass the $id to the view.